tests/Functional/Fixtures/AppKernel.php

```diff
@@ -1,5 +1,7 @@
 <?php
 
+namespace Rokka\RokkaClientBundle\Tests\Functional\Fixtures;
```

Comment:
@Tobion how does this work with both older and newer symfony tests? or should i make this class without namespace and then make composer dev classmap find it?

Reply:
I'd keep it like this and create a `class_alias` for AppKernel to the namespaced class